.github/workflows/continuous-integration-workflow.yml

```diff
@@ -9,10 +9,7 @@ jobs:
     runs-on: ${{ matrix.host }}
     strategy:
       matrix:
-        host: [
-          "ubuntu-latest",
-          "macos-10.15",
-        ]
+        host: ["ubuntu-latest", "macos-10.15"]
```

Comment:
I'd move that beautification to a separate commit to not pollute real meat of the functional change.

Reply:
OK, I will revert code changes, not related to PR.

Reply:
There's no need to revert, just move them to a separate patch to be applied before the bigger functional change in the series.